04-understanding-survey-data-documentation.Rmd

```diff
@@ -229,7 +218,7 @@ library(osfr)
 source("helper-fun/helper-functions.R")
 
 anes_in <- read_rds_tsr("anes_2020.rds") %>%
-  mutate(Weight = V200010b / sum(V200010b) * targetpop)
+  mutate(Weight = V200010b / sum(V200010b) * targetpop) 
```

Comment:
As I mentioned above, we need to explain why we do this.  We don't have this anywhere else.